### PR TITLE
Chat 2: Part 4 - random chat parts

### PR DESCRIPTION
**Pull request by chrisnojima**

### CODE REVIEW

File: shared/chat/conversation/rekey/participant-rekey.js.flow [hunk 1]
```diff
@@ -1,9 +1,8 @@
 // @flow
 import {Component} from 'react'
-import type {RekeyInfo} from '../../../constants/types/chat'
 
 export type Props = {
-  rekeyInfo: ?RekeyInfo,
+  rekeyers: Array<string>,
```
Comment (chrisnojima) [inline shared/chat/conversation/rekey/participant-rekey.js.flow]: components should be dumber

**Comment by chrisnojima** [comment]
@keybase/react-hackers this is ready

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

File: shared/chat/inbox/row/big-teams-divider/index.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+import {ClickableBox, Icon, Box, Badge} from '../../../../common-adapters'
+import {globalStyles, globalColors, globalMargins, glamorous, isMobile} from '../../../../styles'
+import {BigTeamsLabel} from '../big-teams-label'
+
+type Props = {
+  badgeCount: number,
+  toggle: () => void,
+}
+
+const DividerBox = glamorous(Box)({
```
Comment (akalin-keybase) [inline shared/chat/inbox/row/big-teams-divider/index.js]: i thought we're avoiding glamorous?

Comment (chrisnojima) [reply shared/chat/inbox/row/big-teams-divider/index.js]: we are and i changed most of the hot things (rows which mutate and things we scroll around) this is basically a singleton which never changes so leaving it for now. Once chat is in i do want to spend time on making our styling story better so we'll do a larger cleanup then

**Pull request merged**